data_diff/databases/bigquery.py

```diff
@@ -76,6 +76,11 @@ class Dialect(BaseDialect):
     }
     TYPE_ARRAY_RE = re.compile(r"ARRAY<(.+)>")
     TYPE_STRUCT_RE = re.compile(r"STRUCT<(.+)>")
+    # [BIG]NUMERIC, [BIG]NUMERIC(precision, scale), [BIG]NUMERIC(precision)
+    TYPE_NUMERIC_RE = re.compile(r"^((BIG)?NUMERIC)(?:\((\d+)(?:, (\d+))?\))?$")
```

Comment:
[BIG]NUMERIC() not possible?

Reply:
Yep, it's invalid in BQ.
<img width="1840" alt="image" src="https://github.com/datafold/data-diff/assets/18095933/1a869fa4-89ca-4c3c-99f0-b579e0ccb927">